website/app/templates/about.hbs

```diff
@@ -2,7 +2,24 @@
 
 <Doc::Page::Stage>
   <Doc::Page::Cover @title="About" />
-  <Doc::Page::Content @breakthrough={{true}}>
-    <Doc::Cards::Deck @cols="4" @cards={{this.cards.about}} />
+  <Doc::Page::Content>
+    <p class="doc-text-body">Helios is a flexible, scalable design system built on the foundations of accessibility, beautiful design, and consistent user experience.</p>
+    <h2 class="doc-text-h2">Overview</h2>
+    <p class="doc-text-body">The Design Systems Team creates and maintains the Helios Design System. This includes the Figma components, the web components, the documentation, and more. Read through our <a href="/about/overview/" target="_blank" rel="noopener noreferrer">overview</a> for more details, and to see why adopting the HashiCorp Design System is the future for product teams.</p>
+    
```

Comment:
Nit: I would remove the extra "the's" to be less repetitive.

"This includes the Figma components, the web components, the documentation, and more."
=>
"This includes the Figma components, web components, documentation, and more."